src/test/regress/sql/multi_create_fdw.sql

```diff
@@ -12,5 +12,44 @@ RETURNS fdw_handler
 AS 'citus'
 LANGUAGE C STRICT;
 
+set citus.enable_ddl_propagation to off;
```

Comment:
We don't want to propagate the commands between `set citus.enable_ddl_propagation to off;` and `set citus.enable_ddl_propagation to on;`, because these objects are already created on the workers, in `pg_regress_multi.pl`.
We also don't want to remove them since some other tests depend on these.

Reply:
We should probably remove from pg_regress_multi.pl and let this create